src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -83,13 +83,14 @@ protected override async Task<RunResult> RunAsyncInternal(string expr, string se
                 FeatureFlags.StringInterpolation = true;
                 RecalcEngine engine;
                 RecordValue parameters;
+                var iSetup = InternalSetup.Parse(setupHandlerName);
```

Comment:
>InternalSetup.Parse [](http://example.com/codeflow?start=29&length=19)

This is test parsing logic - all backends would need this, but the check here is only in interpreter. 

We should discuss the pattern here.

Reply:
I moved InternalSetup (and related UTs) to Core.Texts\TestRunnerTests